src/main/java/lt/andriaus/hangman/usecase/GameInRamManager.java

```diff
@@ -0,0 +1,36 @@
+package lt.andriaus.hangman.usecase;
+
+import lt.andriaus.hangman.database.Database;
+import lt.andriaus.hangman.domain.Game;
+
+import java.util.*;
+
+public class GameInRamManager implements GameManager{
```

Comment:
kodėl in ram? juk šita klasė nieko nežino apie tavo Database implementaciją. gal vieną kartą paduosi OnFIle ar kokį Postgres implementaciją duombazės